enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -742,21 +749,12 @@ func (p *PebbleCache) Reader(ctx context.Context, d *repb.Digest, offset, limit
 	if err != nil {
 		return nil, err
 	}
-	if p.isolation.GetCacheType() == rfpb.Isolation_ACTION_CACHE {
```

Comment:
Why the remove the remote instance name check?

Reply:
I think it's unnecessary because the remote instance name is stored (well, a crc32 of it) in the file metadata key